### PR TITLE
docs: docstring for types/score

### PR DESCRIPTION
**Pull request by theUnkownName**

### CODE REVIEW

File: jina/types/score/__init__.py [hunk 1]
```diff
@@ -37,18 +37,19 @@ class NamedScore(ProtoTypeMixin):
...
                  copy: bool = False, **kwargs):
-        """
-
-        :param score: the score to construct from, depending on the ``copy``,
-                it builds a view or a copy from it.
-        :param copy: when ``score`` is given as a :class:`NamedScoreProto` object, build a
-                view (i.e. weak reference) from it or a deep copy from it.
-        :param kwargs: other parameters to be set
-        """
+        """Set constructor."""
```
Comment (JoanFM) [inline jina/types/score/__init__.py:52]: why set?

Comment (theUnkownName) [reply jina/types/score/__init__.py:52]: It has to be an imperative sentence. I don't know what else could be better wording, define?

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: jina/types/score/__init__.py [hunk 2]
```diff
@@ -66,19 +67,18 @@ def __init__(self, score: Optional[jina_pb2.NamedScoreProto] = None,
...
+        """Returns computed score between doc ``id`` and ``ref_id``."""
         return self._pb_body.ref_id
 
     @ref_id.setter
     def ref_id(self, val: str):
         self._pb_body.ref_id = val
 
     def set_attrs(self, **kwargs):
-        """Bulk update Document fields with key-value specified in kwargs
+        """Udate Document fields with key-value specified in kwargs.
```
Comment (Yongxuanzhang) [inline jina/types/score/__init__.py:78]: 'Udate' should be' Update' ?

File: jina/types/score/__init__.py [hunk 3]
```diff
@@ -37,18 +37,19 @@ class NamedScore(ProtoTypeMixin):
              score_proto.value = 10.0
              score = NamedScore(score_proto)
 
+    :param score: The score to construct from, depending on the ``copy``,
+        it builds a view or a copy from it.
+    :type score: Optional[jina_pb2.NamedScoreProto]
+    :param copy: When ``score`` is given as a :class:`NamedScoreProto` object, build a
+        view (i.e. weak reference) from it or a deep copy from it.
+    :type copy: bool
+    :param kwargs: Other parameters to be set
```
Comment (Yongxuanzhang) [inline jina/types/score/__init__.py:46]: Missing a period at the end?

**Pull request merged**